test/functorch/test_vmap.py

```diff
@@ -3918,6 +3918,29 @@ def test_vmap_multi_dot_failure_1D_input(self):
         with self.assertRaisesRegex(RuntimeError, "tensor 1 must be 2D but got 1D"):
             return vmap(torch.linalg.multi_dot)(inputs)
 
+    def test_vmap_escaped_error(self):
+        escaped = None
+
+        def f(x):
+            nonlocal escaped
+            escaped = x
+            return x ** 2
+
+        x = torch.randn(3)
+        vmap(f)(x)
+
+        with self.assertRaisesRegex(RuntimeError, r"your tensor may have escaped from vmap.*\(from vmap plumbing\).*"):
+            escaped.sin()
+
+        with self.assertRaisesRegex(RuntimeError, r"your tensor may have escaped from vmap.*\(from boxed inputs\).*"):
+            escaped.sin_()
+
+        with self.assertRaisesRegex(RuntimeError, r"your tensor may have escaped from vmap.*\(from inplace\).*"):
+            escaped.mul_(1)
+
+        vmap(f)(torch.tensor([[0, 0], [0, 0]], dtype=torch.int))
```

Comment:
You can change `torch.tensor([[0, 0], [0, 0]], dtype=torch.int)` to `torch.zeros(2, 2, dtype=torch.int)`

Reply:
```suggestion
        vmap(f)(torch.zeros(2, 2, dtype=torch.int))
```